incubator/sentry-kubernetes/templates/_helpers.tpl

```diff
@@ -0,0 +1,26 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "sentry-kubernetes.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "sentry-kubernetes.fullname" -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/* Generate basic labels */}}
+{{- define "sentry-kubernetes.labels" }}
+app: {{ template "sentry-kubernetes.name" . }}
+heritage: {{.Release.Service }}
+release: {{.Release.Name }}
```

Comment:
Why don't you include the chart label here?

Compare https://github.com/kubernetes/charts/blob/HEAD/incubator/common/templates/_metadata_labels.tpl#L21-L27.